main.go

```diff
@@ -177,6 +179,37 @@ func (a *analyzer) createStoreCommand(
 	}
 }
 
+var hasJSONRE = regexp.MustCompile("--json")
+var hasReportWideRE = regexp.MustCompile("--report-wide")
+func (a *analyzer) mtrCommands() []func() {
+	// Determine what options the machine's mtr offers
+	cmd := exec.Command("mtr", "--help") // nolint: gas, gosec
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		a.storeError(errors.Wrapf(err, "error determining mtr command"))
```

Comment:
I think you want to return here (on error).

Reply:
https://maxmind.slack.com/archives/C8D64BH6D/p1542148423014800

Per discussion on slack, we probably want to include the output along with the error.